views/pages/fullList.ejs

```diff
@@ -0,0 +1,27 @@
+<%- include("../partials/header.ejs", {title: "Game Of Thrones"}) %>
+
+
+<section>
+
+
+  <p>"That's what I do, I drink and I know things" - Tyrion Lannister</p>
+
+  <form action="/fullList" method="GET">
+    <input type="text" name="searchTerm" placeholder="episode search" />
+    <button type="submit">submit</button>
+  </form>
+  <h2><%=filteredEpisodes.length%> of 73 Episodes</h2>
```

Comment:
total episodes was hardcoded so it might not display the correct info with other data (ie. simpsonsData), you're passing through gameOfThronesEpisodes so can do gameOfThronesEpisodes.length if you wanted to :)